.github/workflows/docker.yml

```diff
@@ -25,7 +25,11 @@ jobs:
 
             - uses: hadolint/hadolint-action@v3.1.0
               with:
-                dockerfile: Dockerfile
+                dockerfile: build-tools/docker/slim.Dockerfile
+
+            - uses: hadolint/hadolint-action@v3.1.0
+              with:
+                dockerfile: build-tools/docker/full.Dockerfile
 
             - name: Build image
```

Comment:
I'm building only slim here, I'm wondering whether it make sense to build both. Any opinions?

Reply:
I don't really know. If we're not using too much of the CI time then maybe we should build both versions, but I don't know how to check these limits/the spending

Reply:
I don't think there are limits for public repositories. My concern was about witing for CI, but this job is not always run, so maybe it make sense to run both